.travis.yml

```diff
@@ -7,6 +7,11 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
+  matrix:
+    
+    - CONDA_PY=27
+    - CONDA_PY=34
+    - CONDA_PY=35
```

Comment:
This is not great. Apparently when the feedstock was created this matrix was wrong, but it is now being fixed by the re-rendering. Thoughts as to what happened, @pelson ?

Will pull the bad packages. However, we should combine this with a build number bump to get fresh packages with the Python matrix in effect. Would you be able to do this, @jschueller ?